src/ui/public/stringify/types/Color.js

```diff
@@ -0,0 +1,58 @@
+define(function (require) {
+  return function _StringProvider(Private) {
+    var _ = require('lodash');
+    var FieldFormat = Private(require('ui/index_patterns/_field_format/FieldFormat'));
+
+    require('ui/stringify/editors/color.less');
+
+    _.class(_Color).inherits(FieldFormat);
```

Comment:
I'd like to leave this one for consistency with the other formatters if possible.  It would be a fairly substantial looking change.  Thoughts?